vipps-checkout-how-it-works-woocommerce.md

```diff
@@ -9,6 +9,57 @@ END_METADATA -->
 
 With [Vipps' official WooCommerce plugin](https://wordpress.org/plugins/woo-vipps/), it's easy to start accepting payments with Vipps, VISA or MasterCard through Vipps Checkout.
 
+<iframe width="100%" height="500" src="https://www.youtube.com/embed/86RSKuQ5GME" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
```

Comment:
We can surround it with
<!-- START_HIDDEN_IN_GITHUB

END_HIDDEN_IN_GITHUB -->